definitions/npm/react-i18next_v6.x.x/flow_v0.53.x-/react-i18next_v6.x.x.js

```diff
@@ -0,0 +1,91 @@
+declare module "react-i18next" {
+  declare type TFunction = (key?: ?string, data?: ?Object) => string;
+  declare type Locales = string | Array<string>;
+
+  declare type TranslatorProps = {
+    t: TFunction,
+    i18nLoadedAt: Date,
+    i18n: Object
+  };
+
+  declare type Translator<OP, P> = (
+    component: React$ComponentType<P>
+  ) => Class<React$Component<OP, *>>;
+
+  declare type TranslateOptions = $Shape<{
+    wait: boolean,
+    nsMode: "default" | "fallback",
+    bindi18n: false | string,
+    bindStore: false | string,
+    withRef: boolean,
+    translateFuncName: string,
+    i18n: Object
+  }>;
+
+  declare function translate<OP, P>(
+    locales: Locales,
```

Comment:
locales is an optional argument too ==> https://react.i18next.com/components/translate-hoc.html